tcloud.yml

```diff
@@ -2,7 +2,6 @@ projects:
     public-demo:
         url: https://cloud.tobikodata.com/sqlmesh/tobiko/public-demo/
         gateway: public-demo
-        extras: bigquery,web,github,postgres
-        pip_executable: uv pip
```

Comment:
This should be allowed to work for users as most tcloud customers are using uv pip today and it's much better experience for them.